workflows/images/cellranger/README.md

```diff
@@ -0,0 +1,39 @@
+This folder contains a Dockerfile for the cellranger analysis.
+
+## Building the image
+
+In order to build this image, the cellranger archive component must be downloaded separately to comply with licensing, and should be placed in this folder.
+The current version of this file is `cellranger-4.0.0.tar.gz` and can be downloaded from [10X Genomics Website](https://support.10xgenomics.com/single-cell-gene-expression/software/downloads/latest) after agreeing to their license terms.
+
+Following download of cellranger, you can build the image with:
+```
+docker build . -t scpca-cellranger:4.0.0
+```
+
+At this point, the image should be ready for use on the local machine.
+
+## Pushing to AWS ECR
+If the updated image needs to be pushed to AWS ECR, you can follow the steps below to push to the Alex's Lemonade ECR repository.
+
+First, login to the AWS ECR docker repository with:
+```
+aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin 589864003899.dkr.ecr.us-east-1.amazonaws.com
+```
+Note that for the code above to work, you must have set up `aws` command line tools on your machine, and have run [`aws configure`](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html) to set up your credentials for AWS access.
+This is a private repository; access is only available to Alex's Lemonade (CCDL) users at this time.
+
+Following login (which only needs to be done once), tag and push the docker image with:
+
+```
+docker tag scpca-cellranger:4.0.0 589864003899.dkr.ecr.us-east-1.amazonaws.com/scpca-cellranger:latest
+docker tag scpca-cellranger:4.0.0 589864003899.dkr.ecr.us-east-1.amazonaws.com/scpca-cellranger:4.0.0
+docker push 589864003899.dkr.ecr.us-east-1.amazonaws.com/scpca-cellranger:4.0.0
+docker push 589864003899.dkr.ecr.us-east-1.amazonaws.com/scpca-cellranger:latest
```

Comment:
I think I pushed a docker image successfully! 

Next question is do we prefer tagging with `:4.0.0` or `:latest`? Might be easier to choose one.

Reply:
We want both; if we update cellranger, we still want something tagged with `latest`. And don't push over my image! (Hopefully you got a message that the image was already present?)

Reply:
It said some layers were already present, but it still seems like it pushed something. I don't have the log message anymore.

Reply:
I have clarified that this is not something people should do in most cases... Switched order to emphasize `pull` over `push`.